client-app/shared/catalog/components/product-price-block.vue

```diff
@@ -122,6 +134,8 @@ function addToList() {
   });
 }
 
+const mailToLink = computed(() => `mailto:?subject=${props.product?.name}&body=${pageUrl}`);
```

Comment:
Product name should be escaped. Special chars like `&` or `+` are not handled properly.
Both subject and body should be wrapped in `encodeURIComponent` to accept any string.